src/components/Nav/Nav.js

```diff
@@ -45,13 +45,20 @@ const NavBar = styled.nav`
   margin: 0;
   padding: 0;
   background-color: black;
-  border-top: 1px solid white;
+
   display: flex;
   flex-direction: row;
   justify-content: space-around;
   align-items: center;
-  height: 12vh;
+  height: 10vh;
   position: sticky;
   bottom: 0;
 `
+const NavigationLink = styled(NavLink)`
+  color: grey;
+  &.${props => props.activeClassName} {
+    color: white;
+    height: 7vh;
+  }
```

Comment:
I think it would work the same if you add the class here: 
```suggestion
  &.is-active {
    color: white;
    height: 7vh;
  }
```